test/NuGet.Core.Tests/NuGet.Common.Test/PathResolverTests.cs

```diff
@@ -664,17 +678,20 @@ public void Dispose()
         private void PopulateTestDirectory()
         {
             var rootDirectory = new DirectoryInfo(_rootDirectory.Path);
-            DirectoryInfo directory1 = Directory.CreateDirectory(System.IO.Path.Combine(rootDirectory.FullName, "dir1"));
-            DirectoryInfo directory2 = Directory.CreateDirectory(System.IO.Path.Combine(directory1.FullName, "dir2"));
-            DirectoryInfo directory3 = Directory.CreateDirectory(System.IO.Path.Combine(directory2.FullName, "dir3"));
-            DirectoryInfo directory4 = Directory.CreateDirectory(System.IO.Path.Combine(directory1.FullName, "dir4"));
-            Directory.CreateDirectory(System.IO.Path.Combine(rootDirectory.FullName, "dir5"));
+            var directory1 = Directory.CreateDirectory(System.IO.Path.Combine(rootDirectory.FullName, "dir1"));
+            var directory2 = Directory.CreateDirectory(System.IO.Path.Combine(directory1.FullName, "dir2"));
+            var directory3 = Directory.CreateDirectory(System.IO.Path.Combine(directory2.FullName, "dir3"));
+            var directory4 = Directory.CreateDirectory(System.IO.Path.Combine(directory1.FullName, "dir4"));
+            var directory5 = Directory.CreateDirectory(System.IO.Path.Combine(rootDirectory.FullName, "dir5"));
+            var directory6 = Directory.CreateDirectory(System.IO.Path.Combine(rootDirectory.FullName, "dir6"));
 
             CreateTestFiles(rootDirectory);
             CreateTestFiles(directory1);
             CreateTestFiles(directory2);
             CreateTestFiles(directory3);
             CreateTestFiles(directory4);
+
+            File.WriteAllText(System.IO.Path.Combine(directory6.FullName, "FILE3.EMPTY"), string.Empty);
```

Comment:
`System.IO` is redundant because this namespace is mentioned in the `using` statement on top of the file.

Reply:
Same as https://github.com/NuGet/NuGet.Client/pull/2901/files#r389479408